packages/ui/src/2_molecules/Dialog/Dialog.module.css

```
@@ -12,21 +10,12 @@
 }
 
 .dialog {
-  @apply relative bg-black text-sov-white px-4 py-8 mx-auto min-h-full;
+  @apply relative bg-sovryn-black text-sov-white p-0 mx-auto min-h-full;
```

```suggestion
  @apply relative bg-sovryn-black text-gray-10 p-0 mx-auto min-h-full;
```
I think it should be `gray-10` according to the Figma.